ext/standard/tests/file/bug22414.phpt

```diff
@@ -1,5 +1,11 @@
 --TEST--
 Bug #22414 (passthru() does not read data correctly)
+--SKIPIF--
+<?php
+if (str_contains(getenv('TEST_PHP_EXECUTABLE'), " ")) {
```

Comment:
the test even handles linux/Win differently, why it cannot be escaped?

Reply:
I had a brain error, I *think* I figured it out and in the process managed to get rid of the two different code paths. Let's see...